### PR TITLE
OCPBUGS-1598: updates toast wording to Workload

### PR DESCRIPTION
**Pull request by invincibleJai**
Tracks: https://issues.redhat.com/browse/OCPBUGS-1598

### CODE REVIEW

**Review by jerolimov — CHANGES_REQUESTED** [review]
Works fine, but I have two concerns I would recommend to improve or check with @serenamarie125 or UX if we want continue with the current solution.

**a) The `Route` link in the toast is not shown for Knative Services.**

Sure, it\s an improvement to show the modal, but we added this toast to make it easier to open the application (component) directly.

And with defaulting to Serverless Services we should also show links for this.

One implementation option might be to add a watcher in `ImportToastContent` if the `deployedResource` (see also below) is a Knative `Service`.

**b) Term workload**

Also if I added this suggestion to the issue, I'm unsure if we w use the term Workload here or if we should rephrase it to:

`Resource created successfully.`

or:

`Component created successfully.`

or IMHO even better:

`${resourceKind} ${resourceName} created successfully.`

From the implementation point of view this should be possible if we just pass one `deployedResource` to `ImportToastContent` and pass the first Deployment / DeploymentConfig or Knative Service. (Changing `resources.filter` to `resources.find`)

Wdyt?

**Comment by invincibleJai** [comment]
> Works fine, but I have two concerns I would recommend to improve or check with @serenamarie125 or UX if we want continue with the current solution.
> 
> **a) The `Route` link in the toast is not shown for Knative Services.**
> 
> Sure, it\s an improvement to show the modal, but we added this toast to make it easier to open the application (component) directly.
> 
> And with defaulting to Serverless Services we should also show links for this.
> 
> One implementation option might be to add a watcher in `ImportToastContent` if the `deployedResource` (see also below) is a Knative `Service`.

Yeah I was hoping this can be handled with a separate issue as ksvc route is not like the k8s route which we create as resources with import flow for Deployment and DC so will need some more work on it. Let's discuss on it.

> 
> **b) Term workload**
> 
> Also if I added this suggestion to the issue, I'm unsure if we w use the term Workload here or if we should rephrase it to:
> 
> `Resource created successfully.`
> 
> or:
> 
> `Component created successfully.`
> 
> or IMHO even better:
> 
> `${resourceKind} ${resourceName} created successfully.`
> 
> From the implementation point of view this should be possible if we just pass one `deployedResource` to `ImportToastContent` and pass the first Deployment / DeploymentConfig or Knative Service. (Changing `resources.filter` to `resources.find`)
> 
> Wdyt?

We can do whatever way we would like to go in current logic the check is to see if the deployed resource is one or more, am not sure in which scenario it'll be more (in the current import flow) so we can show it as above for the case when the deployed resource is 1.

**Review by jerolimov — APPROVED** [review]
Tested this locally and it works fine for Deployments, DeploymentConfigs and Knative Services.

/lgtm

**Review by vikram-raj — COMMENTED** [review]
/lgtm
